services/headless-lms/migrations/20210816123758_course_localization_support.up.sql

```diff
@@ -0,0 +1,33 @@
+-- Create new table for course settings
+CREATE TABLE user_course_settings (
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+  deleted_at TIMESTAMP WITH TIME ZONE,
+  user_id UUID NOT NULL REFERENCES users(id),
+  course_id UUID NOT NULL REFERENCES courses(id),
+  current_course_instance_id UUID NOT NULL REFERENCES course_instances(id),
+  UNIQUE (user_id, course_id)
```

Comment:
FOREIGN KEY (b, c) REFERENCES other_table (c1, c2)

Reply:
Will be done in the second part of the task.